Examples fail to start: rollup command not recognized

Reproduce on windows node 10, yarn 1.15.2:
* Forked: https://github.com/FormidableLabs/urql/tree/master/examples/1-getting-started
* yarn
* yarn start

Output:
```
$ yarn start
yarn run v1.15.2
$ npm run build --prefix ../../

> urql@1.1.1 prebuild C:\development\urql
> rimraf dist


> urql@1.1.1 build C:\development\urql
> run-p build:types build:bundle


> urql@1.1.1 build:bundle C:\development\urql
> rollup -c rollup.config.js


> urql@1.1.1 build:types C:\development\urql
> tsc -d --emitDeclarationOnly --outDir dist/types

'rollup' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! urql@1.1.1 build:bundle: `rollup -c rollup.config.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the urql@1.1.1 build:bundle script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\cooke\AppData\Roaming\npm-cache\_logs\2019-06-14T11_32_29_226Z-debug.log
ERROR: "build:bundle" exited with 1.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! urql@1.1.1 build: `run-p build:types build:bundle`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the urql@1.1.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\cooke\AppData\Roaming\npm-cache\_logs\2019-06-14T11_32_29_878Z-debug.log
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/examples/1-getting-started/README.md
+++ b/examples/1-getting-started/README.md
@@ -1,21 +1,36 @@
 ## About
 
-This is a basic example of subscriptions being used with URQL.
+This is a basic todo app built with `urql`. It demonstrates some of the core principles of the framework, including:
+
+- Creating a `client`.
+- Wrapping your app with `Provider`.
+- Using the `useQuery` hook to execute and refetch queries.
+- Using the `useMutation` hook to execute mutations.
 
 ## Usage
 
-#### 1. Install dependencies
+#### 1. Install dependencies in repo root.
+
+To get started with the example, make sure to install `urql`'s dependencies in the root directory of your clone. We `link` to the `urql` source from the `examples` directory, so its dependencies need to be installed.
 
 ```bash
+# In root directory
 yarn
 ```
 
-#### 2. Start server
+#### 2. Navigate to this directory and install dependencies.
+
+```bash
+cd examples/1-getting-started
+yarn
+```
+
+#### 3. Start the example app and server (this is done in a single command).
 
 ```bash
 yarn start
 ```
 
-#### 3. Open browser
+#### 4. Open your browser
 
 Navigate to example at [http://localhost:3000/](http://localhost:3000/).

--- a/examples/1-getting-started/src/app/Home.tsx
+++ b/examples/1-getting-started/src/app/Home.tsx
@@ -1,13 +1,16 @@
-import React, { FC, useCallback } from 'react';
+import React, { FC, useCallback, useMemo } from 'react';
 import gql from 'graphql-tag';
 import { useQuery } from 'urql';
 import { Error, Loading, Todo } from './components';
 
+interface ITodo {
+  id: string;
+  text: string;
+  complete: boolean;
+}
+
 interface QueryResponse {
-  todos: Array<{
-    id: number;
-    text: string;
-  }>;
+  todos: ITodo[];
 }
 
 export const Home: FC = () => {
@@ -17,7 +20,7 @@ export const Home: FC = () => {
     []
   );
 
-  const getContent = () => {
+  const todos = useMemo(() => {
     if (res.fetching || res.data === undefined) {
       return <Loading />;
     }
@@ -28,16 +31,16 @@ export const Home: FC = () => {
 
     return (
       <ul>
-        {res.data.todos.map(todo => (
+        {res.data.todos.map((todo: ITodo) => (
           <Todo key={todo.id} {...todo} />
         ))}
       </ul>
     );
-  };
+  }, [res]);
 
   return (
     <>
-      {getContent()}
+      {todos}
       <button onClick={refetch}>Refetch</button>
     </>
   );

--- a/examples/1-getting-started/src/app/components/Todo.tsx
+++ b/examples/1-getting-started/src/app/components/Todo.tsx
@@ -1,18 +1,26 @@
-import React from 'react';
+import React, { FC } from 'react';
 import { useMutation } from 'urql';
 
-export const Todo = props => {
+interface Props {
+  complete: boolean;
+  text: string;
+  id: string;
+}
+
+export const Todo: FC<Props> = props => {
   const [mutation, executeMutation] = useMutation(RemoveTodo);
 
   const handleToggle = () => executeMutation({ id: props.id });
 
   return (
     <li onClick={handleToggle}>
-      <p className={`${props.complete ? 'strikethrough' : ''}`}>{props.text}</p>
+      <p className={props.complete ? 'strikethrough' : ''}>{props.text}</p>
       {mutation.fetching && <span>(updating)</span>}
     </li>
   );
 };
+
+Todo.displayName = 'Todo';
 
 const RemoveTodo = `
 mutation($id: ID!) {

--- a/examples/1-getting-started/src/app/index.css
+++ b/examples/1-getting-started/src/app/index.css
@@ -35,6 +35,7 @@ li {
   padding: 20px;
   margin: 20px 0;
   border-radius: 4px;
+  cursor: pointer;
 }
 
 li span {

--- a/examples/1-getting-started/src/app/index.tsx
+++ b/examples/1-getting-started/src/app/index.tsx
@@ -1,7 +1,7 @@
 import React, { FC } from 'react';
 import * as ReactDOM from 'react-dom';
 import { createClient, Provider } from 'urql';
-import { Home } from './home';
+import { Home } from './Home';
 import './index.css';
 
 const client = createClient({

